app/controllers/reactions_controller.rb

```diff
@@ -31,6 +31,10 @@ def find_or_initialize_reaction
   end
 
   def redirect_url
-    params[:redirect_url].presence || course_lesson_path(@reaction.course_slug, @reaction.lesson_slug)
+    if params[:redirect_url].present? && params[:redirect_url].starts_with?('https://soulmedicine.io/')
```

Comment:
Why is the alert still showing up when the code that's in the `redirect_url` method has changed (for the better) (and the "paths" section knows that it has changed)?

Reply:
Would there be any benefit to using `URI` instead of `String#start_with?`

```suggestion
    if params[:redirect_url].present? && URI(params[:redirect_url]).host == 'soulmedicine.io'
```